### PR TITLE
React js

### PR DESCRIPTION
**Pull request by DavidAnatolie**
Screen Shot 2020-05-26 at 2 44 39 PM
#2 
The purpose of this ticket is to setup ReactJS using create-react-app. I tested it by running npm start.

### CODE REVIEW

File: README.old.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# DDS
+
+One Paragraph of project description goes here
```
Comment (danielanatolie) [inline README.old.md:3]: 2 Readmes?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (danielanatolie) [inline .gitignore:1]: Ensure you have only a single .gitignore after merging BE code into master.

Comment (DavidAnatolie) [reply .gitignore]: https://github.com/DavidAnatolie/DDS/pull/6/commits/f7d3e567062e2c645863e1820e56283da8047abc

File: README.old.md (hunk 1 above)
Comment (DavidAnatolie) [reply README.old.md]: f7d3e56

**Comment by DavidAnatolie** [comment]
Screen Shot 2020-05-28 at 10 20 42 AM

f7d3e56

**Pull request merged**